src/compiler/program.ts

```diff
@@ -673,8 +851,8 @@ namespace ts {
             return getBaseFileName(fileName).indexOf(".") >= 0;
         }
 
-        function processRootFile(fileName: string, isDefaultLib: boolean) {
-            processSourceFile(normalizePath(fileName), isDefaultLib);
+        function processRootFile(fileName: string, isDefaultLib: boolean, supportedExtensions: string[]) {

```

Comment:
Let me look this up, this code keeps changing so much that I don't know if we really need this any more.